shared/enums.proto

```diff
@@ -67,6 +67,11 @@ enum EventType {
     SPECTRAL_LINE_RESPONSE = 68;
     CONCAT_STOKES_FILES = 69;
     CONCAT_STOKES_FILES_ACK = 70;
+    STOP_FILE_LIST = 71;
```

Comment:
@markccchiang can we have a single message with a file type parameter in order to cancel either the image file list or the catalog file list.